packages/mobx-state-tree/src/utils.ts

```diff
@@ -51,7 +51,7 @@ export function extendKeepGetter(a: any, ...b: any[]) {
     for (let i = 0; i < b.length; i++) {
         const current = b[i]
         for (let key in current) {
-            const descriptor = Object.getOwnPropertyDescriptor(current, key)
```

Comment:
This fixes the error `src/utils.ts(55,26): error TS2532: Object is possibly 'undefined'.`